recipe/meta.yaml

```diff
@@ -10,6 +10,8 @@ build:
     - COPY "%RECIPE_DIR%\\run_conda_forge_build_setup_win" "%SCRIPTS%\\run_conda_forge_build_setup.bat"                  # [win]
     - cp "${RECIPE_DIR}/run_conda_forge_build_setup_osx" "${PREFIX}/bin/run_conda_forge_build_setup"                     # [osx]
     - cp "${RECIPE_DIR}/run_conda_forge_build_setup_linux" "${PREFIX}/bin/run_conda_forge_build_setup"                   # [linux]
+    - COPY "%RECIPE_DIR%\\upload_or_check_non_existence.py" "%SCRIPTS%\\upload_or_check_non_existence.py"                # [win]
+    - cp "${RECIPE_DIR}/upload_or_check_non_existence.py" "${PREFIX}/bin/upload_or_check_non_existence.py"               # [osx]
```

Comment:
Please see the fix in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/35 ).